package/lite/etc/conf.d/sources/source_syslog/plugin.jinja

```diff
@@ -1,414 +0,0 @@
-
```

Comment:
it's a good idea to reuse existing jinja. Did you review changes before switching the main version?
`diff package/lite/etc/conf.d/sources/source_syslog/plugin.jinja package/etc/conf.d/sources/source_syslog/plugin.jinja`

Reply:
In this template we don't using all the topics, as I mentioned [here](https://github.com/splunk/splunk-connect-for-syslog/pull/2242#discussion_r1362103385) we will use all topics (like almost-syslog, cef, leef), because pluggable modules an local parser requrying them